x/fundraising/spec/02_state.md

```diff
@@ -46,7 +46,12 @@ type AuctionI interface {
 
 	GetVestingSchedules() []VestingSchedule
 	SetVestingSchedules([]VestingSchedule) error
-
+	
+	GetWinningPrice() sdk.Dec  // To load WinningPrice at the current time
+	GetNumberWinningBidders() uint64
+	
+	GetRemainingSellingCoin() sdk.Coin // To calculate RemainingSellingCoin
```

Comment:
Is there a reason why you don't have Setter functions?

Reply:
I will add the setters.